harness/go/harness/util.go

```diff
@@ -13,3 +17,31 @@ var RetryDisabled = &temporal.RetryPolicy{MaximumAttempts: 1}
 func AppErrorf(msg string, args ...interface{}) error {
 	return temporal.NewApplicationError(fmt.Sprintf(msg, args...), "SdkFeaturesError")
 }
+
+type JSONPayload struct {
+	Data     string            `json:"data"`
+	Metadata map[string]string `json:"metadata"`
+}
+
+func UnmarshalPayload(data []byte) (*JSONPayload, error) {
+	var payload JSONPayload
+	return &payload, json.Unmarshal(data, &payload)
+}
+
+// Thanks stackoverflow, I hope this is acceptable
```

Comment:
Can remove these helpers based on my comment above. We provide the absolute directory via the runner feature.

Reply:
👍